generic_config_updater/gu_common.py

```diff
@@ -151,6 +147,24 @@ def remove_empty_tables(self, config):
                 config_with_non_empty_tables[table] = copy.deepcopy(config[table])
         return config_with_non_empty_tables
 
+    def create_sonic_yang_with_loaded_models(self):
+        # sonic_yang_with_loaded_models will only be initialized once the first time this method is called
+        if self.sonic_yang_with_loaded_models is None:
+            loaded_models_sy = sonic_yang.SonicYang(self.yang_dir)
+            loaded_models_sy.loadYangModel()
+            self.sonic_yang_with_loaded_models = loaded_models_sy
+
+        # create a copy of sonic_yang_with_loaded_models, only copy the loaded yang models related properties
```

Comment:
>create [](http://example.com/codeflow?start=10&length=6)

This is similar to shallow copy. Did you consider [https://docs.python.org/3/library/copy.html#copy.copy](https://docs.python.org/3/library/copy.html#copy.copy)?

Reply:
Good idea